src/sas/qtgui/Plotting/Slicers/AnnulusSlicer.py

```diff
@@ -104,9 +104,11 @@ def _post_data(self, nbins=None):
                    numpy.fabs(self.outer_circle.get_radius()))
         # If the user does not specify the numbers of points to plot
         # the default number will be nbins= 36
-        if nbins is None:
-            self.nbins = 36
-        else:
+#        if nbins is None:
+#            self.nbins = 20
+#        else:
+#            self.nbins = nbins
+        if nbins is not None:
```

Comment:
Is this commented section supposed to remain commented out? What happens if the `nbins` argument is None (which is the default)?

Reply:
woops ... good catch. Actually the comments just above the section are also wrong now. I will remove all those comments. 

Regarding what happens if `nbins = None`, it will be reset to the default value set at the beginning by the validate method.  That was not the case for the box slicer but it is now :-)